README.md

```diff
@@ -12,14 +12,10 @@ Clone this repo.
 
 ## Usage
 
-To repeat the preprocessing steps download the following files to `data/raw/`
+To repeat the preprocessing steps download the following file to `data/raw/`
```

Comment:
I know I wrote this originally, but could we add ` +To repeat the preprocessing steps, please download the following file to `data/raw/` first` :)